seps/sep-0002.md

```diff
@@ -0,0 +1,169 @@
+---
+SEP: 2
+Title: WIP Standardize Groups
+Authors: Charles G. <charles@near.org>
+Status: Draft
+DiscussionsTo: https://github.com/nearsocial/SEPs/pull/2
+Version: 1.0.0
+Created: 2023-10-15
+Last Updated: 2023-11-21
+---
+
+
+## Summary
+
+This document serves to come to an agreement on a specification for a new "Groups" experience on Near Social, designed to allow people to create and discover communities around specific topics, interests, projects, governance, etc.
+
+## Motivation
+
+We need a SocialDB standard for groups to support discoverability on near.org and interoperability between gateways and dApps to mitigate the [compatibility problems](https://near.social/devgovgigs.near/widget/gigs-board.pages.Post?id=1034) which arise when dApps build distinct implementations of communities.
+
+
+## UX Goals
+
+1. To allow a group of users to moderate with sovereignty a scope of socialDB limited to the members of the group. e.g. group members have read + write access to the group's socialDB namespace. 
+1. Non-members only have read-only access to view group content
+1. Group members should receive notifications for all relevant social interactions in a group (e.g. Like, Reply for Posts and Discussions)
+1. All content posted to groups that a user is part of should appear the user’s "Following" activity feed. 
+1. Leaving a group will stop content from appearing on a user’s feed.
+1. Group content should only appear in the “All” activity feed if a group member reposts a group post to the main feed.
+1. One should be able to retrieve a list of all groups in socialDB
+1. One should be able to retrieve a list of all users in a group.
+1. One should be able to search for groups, where search text is matched against the group's name and metadata.
+
+
+## Rationale and Alternatives
+
+A key success metric of this standard is to provide a minimal blueprint that gateways can build upon to ensure feature parity, in the UX goals above, across gateways. 
+
+Prior art: 
+
+1. near.social's NFT-based groups
+
+The [group's post page](https://near.org/near/widget/ComponentDetailsPage?src=mob.near/widget/N.Group&tab=source); passing to this page an NFT's contractID as the groupId query param will [render the group details](https://near.org/mob.near/widget/N.Group?groupId=v0.8bityonce.near).   A primary drawback to this NFT-based approach is that there is no canonical list of group members. Instead, group membership is inferred from the owners of a NFT per a non-socialDB API - a Mintbase API is used in the above component. 
```

Comment:
Would that mean that every NFT contract is a group or should we put some restrictions?

Reply:
This is a great idea 👍